### DISCUSSION TITLE
Executing DMV?

### DISCUSSION THREAD

**Discussion by SharmaAntriksh** [TabularEditor/TabularEditor3#802] [Other Topics]
I was wondering is there a way to execute a DMV inside the C# editor? I am trying to list dependent objects and then perform some actions. I tried to import using Microsoft.AnalysisServices.AdomdClient; but it doesn't work. Any ideas?

**Comment by pawarbi** [comment]
I think `ExecuteDax("SELECT * FROM $SYSTEM.YOUR_DMV_NAME").Tables[0];` is what you are looking for.
Kurt Buhler (Data Goblins) also has a blog post on it.
